kubernetes/cray-opa/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-opa
-version: 1.32.1
+version: 1.32.2
```

Comment:
I feel like this might be worth a minor bump.

Reply:
I followed similar moves forward based on what other recently merged PRs to this project had done.